app/assets/stylesheets/components/page_specific/_link_list.scss

```diff
@@ -0,0 +1,44 @@
+/*
+  Link list component
+  Currently used on category pages to list out yellow underlined links
+*/
+.link-list {
+  @include body(22, 40);
+  font-weight: bold;
+
+  .l-category--image & {
+    @include respond-to($mq-m) {
+      width: 60%;
+    }
+  }
+}
+
+.link-list__item {
+  display: inline;
+  margin-right: $baseline_unit;
+
+  &:before {
+    // This is to override the highly specific .editorial list styling
+    content: none !important;
+  }  
+}
+
+.link-list__item:after {
+  content: ", ";
+}
+ 
+.link-list__item:last-child:after {
+  content: none;
+}
+
+.link-list__item__link {
+  color: $color-true-black !important;
```

Comment:
Can we use `:link` style states to negate need for !important here? (I can't remember if we spoke about this the other day?)